operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -231,8 +232,23 @@ public void reportSuspendedFibers() {
             gate.getCurrentFibers().forEach(
                 (key, fiber) -> {
                   Optional.ofNullable(fiber.getSuspendedStep()).ifPresent(suspendedStep -> {
-                    LOGGER.fine("Namespace: " + namespace + ", DomainUid: " + key
-                        + ", Fiber: " + fiber.toString() + " is SUSPENDED at " + suspendedStep.getName());
+                  
+                    // find out the domainUID 
+                    Packet packet = fiber == null ? null : fiber.getPacket();
+                    String domainUid = null;
+                    if (packet != null) {
+                      DomainPresenceInfo info = packet.getSpi(DomainPresenceInfo.class);
+                      if (info != null) {
+                        domainUid = info.getDomainUid();
+                      }
+                    }
+                    LoggingContext.context(new LoggingContext().namespace(namespace).domainUid(domainUid));
```

Comment:
consider adding a utility method in LoggingContext that handles setting the context, invoke the code, and removal of context, something like `LoggingContext.runUnderContext(LoggingContext, Runnable)`.

Reply:
Or maybe have a Closable wrapper to allow for a 'try (closable) {}' pattern?

Reply:
Changed to use AutoCloseable.